assets/sass/components/idea-hub/_googlesitekit-idea-hub-wpdashboard-widget.scss

```diff
@@ -42,6 +42,7 @@
 
 	.googlesitekit-idea-hub__wpdashboard--copy {
 		color: map-get($wp-colors, "blue-80");
+		margin-right: 14px;
```

Comment:
This looks great but it ends up pushing the text to two lines on a 4-col dashboard (en-US of course).

![image](https://user-images.githubusercontent.com/1621608/132199272-83e58ed4-deae-4b9f-acba-0ed87cc00da7.png)

@marrrmarrr how about we change the CTA to "View Ideas" instead?

![image](https://user-images.githubusercontent.com/1621608/132199418-8ee4e4a4-c1ee-4122-a37d-c51092238b70.png)

Reply:
@johnPhillips, @marrrmarrr confirmed that this change is okay to make. Would you please update this as well?

Reply:
Of course, good idea 👍